lib/date.js

```diff
@@ -48,6 +48,38 @@ function duration (string, number, unit) {
   }
 }
 
+/**
+ * Returns the number of days ago from the given date.
+ * Today’s date will return 0.
+ * Dates in the future return a negative number
+ * Where times are given, the number of days is rounded down.
+ *
+ * @example <caption>Yesterday</caption>
+ * daysAgo('2023-06-15', 1) //
+ *
+ * @example <caption>Today</caption>
+ * daysAgo('2023-06-16', 0) //
+ *
+ * @example <caption>Two week’s in future</caption>
+ * daysAgo('2023-06-30', -14) //
+ *
+ * @param {string} string - Date
+ * @returns {number} number representing how many days ago it was
+ */
+function daysAgo (string) {
+  const timeZone = 'Europe/London'
+
+  string = normalize(string, '')
+  try {
+    const date = DateTime.fromISO(string, { zone: timeZone })
+    const today = DateTime.now().setZone(timeZone)
+
+    return Math.floor(today.diff(date, 'days').toObject().days)
```

Comment:
I did start out by using the native Date objects, with `Date.parse()` and `Math.floor((today.getTime() - date.getTime()) / (1000 * 3600 * 24))` which worked fine, but decided to switch to the luxon objects for consistency with the other methods, and better error handling if an invalid date string is given.